ibis/tests/all/test_generic.py

```diff
@@ -66,6 +67,7 @@ def test_coalesce(backend, con, expr, expected):
         assert result == expected
 
 
+@pytest.mark.skip_backends((MSSQL,))
```

Comment:
any reason to use skip_backend here instead of xfail_backends?

Reply:
no reason at all. changed to xfail_backends